pkg/agent/config/node_config.go

```diff
@@ -127,6 +131,8 @@ type NodeConfig struct {
 	UplinkNetConfig *AdapterNetConfig
 	// The config of the WireGuard interface.
 	WireGuardConfig *WireGuardConfig
+	// The config of the Egress interface.
```

Comment:
nit: there is a WireGuard interface but there is no Egress interface. Maybe "The config of the Egress feature."

Reply:
OK, I will fix it later.